cmd/ctr/utils_unix.go

```diff
@@ -21,8 +21,8 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
-func prepareStdio(stdin, stdout, stderr string, console bool) (*sync.WaitGroup, error) {
-	var wg sync.WaitGroup
+func prepareStdio(stdin, stdout, stderr string, console bool) (wg *sync.WaitGroup, err error) {
+	wg = &sync.WaitGroup{}
```

Comment:
You can't put `wg` into a `var` block here because then it shadows the named return value. Assuming you meant something like (incremental on this PR):
```diff
diff --git a/cmd/ctr/utils_unix.go b/cmd/ctr/utils_unix.go
index b7430af..71eb15d 100644
--- a/cmd/ctr/utils_unix.go
+++ b/cmd/ctr/utils_unix.go
@@ -22,10 +22,13 @@ import (
 )
 
 func prepareStdio(stdin, stdout, stderr string, console bool) (wg *sync.WaitGroup, err error) {
-	wg = &sync.WaitGroup{}
-	ctx := gocontext.Background()
+	var (
+		wg  = &sync.WaitGroup{}
+		ctx = gocontext.Background()
+		f   io.ReadWriteCloser
+	)
 
-	f, err := fifo.OpenFifo(ctx, stdin, syscall.O_WRONLY|syscall.O_CREAT|syscall.O_NONBLOCK, 0700)
+	f, err = fifo.OpenFifo(ctx, stdin, syscall.O_WRONLY|syscall.O_CREAT|syscall.O_NONBLOCK, 0700)
 	if err != nil {
 		return nil, err
 	}

```

I could move `f` and `ctx` and leave the `wg = &sync.WaitGroup{}` but I'm not sure there is much benefit and it's getting a bit of topic for this PR (but just say the word and I'll do it since I'm in here anyway).